Oloughlin_Transmissivity_Calculation

```diff
@@ -282,18 +265,22 @@ def Oloughlin_graph(Psumpts, Qsumpts, baseflow):
     OData = pd.DataFrame()
     OData['1/Qo'] = figdata['1/Qo'] 
     OData['0-1'] = (figdata['Q/P Points'])
-    
+    #filtering data for runoff events that are not infinity, greater than 1, or less than tolerance 0.0002
+    OData = OData[OData['0-1'] >= 0.0002]
+    OData = OData[OData['0-1'] != (-math.inf)]
+    OData = OData[OData['0-1'] != 1]
+    OData = OData.reset_index()
     return OData
 
-def transmissivity_calc(ti, OData):
+def transmissivity_calc(ti, OData, titlename ='Elder Creek'):
     """
 
     Parameters
     ----------
     ti : Pandas Dataframe
-        Columns left to right: TI (ti values sorted from highest to lowest, float), 0-1 (values starting at 1 on the 0 index TI value and going down to 0 on the last value)
+        Columns left to right: TI (ti values sorted from highest to lowest, float), '0-1' (values starting at 1 on the 0 index TI value and going down to 0 on the last value)
```

Comment:
I might have a tendency to put all column names that are strings in quotes, both here and in other functions. 'TI' etc. For example look at how they write the parameters for xscale: https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.xscale.html